runtime/libffi/z/sysvz64.s

```diff
@@ -35,25 +35,22 @@ FFISYS CELQPRLG DSASIZE=DSASZ,PSECT=ASP
 
          LGR 5,14              Copy of parameter types
 *Reset registers used in code gen
-*         SR  0,0              GPR counter
-          LA  0,0
-*         SR  14,14            FPR counter
-          LA  14,0
-*         SR  7,7              Offset in the arg area
-          LA  7,0
-*         SR  10,10            Offset in array of parm types
-          LA  10,0
-*         SR  6,6              Offset in stored parm values
-          LA  6,0
+          LA  0,0             GPR counter
+          LA  14,0            FPR counter
+          LA  7,0             Offset in the arg area
+          LA  10,0            Offset in array of parm types
+          LA  6,0             Offset in stored parm values
```

Comment:
The opcodes and comments should line up with those on line 36 (or revert this block of changes altogether).

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/compare/583867825be4477a8319f21ffb952780631fbeb3..f6eb07952b7a7b7c8c8789fbb19e2c66f7672683

Reply:
Not done: to match surrounding code, there should be two more spaces before each comment:
```
         LA  0,0               GPR counter
         LA  14,0              FPR counter
         LA  7,0               Offset in the arg area
         LA  10,0              Offset in array of parm types
         LA  6,0               Offset in stored parm values
```